sdk/communication/communication-phone-numbers/review/communication-phone-numbers.api.md

```diff
@@ -109,7 +112,10 @@ export interface PurchasedPhoneNumber {
     cost: PhoneNumberCost;
     countryCode: string;
     id: string;
+    operatorId: string;
+    operatorName: string;
```

Comment:
I thought we'd wrap these into a separate object.

Reply:
After the review discussions, the current way forward is to use this version for private preview, to unblock the UI and testing. We'll revisit the comments together with API for public preview and work-in the comments from review.

Reply:
ok, thanks